### PR TITLE
SLE12 release notes

### PR DESCRIPTION
**Pull request by dzedro**
GNOME:
http://10.100.98.90/tests/1019/modules/releasenotes/steps/1/src
SDK+HA:
http://10.100.98.90/tests/1018/modules/releasenotes/steps/1/src
SDK+HA+GEO
http://10.100.98.90/tests/1021/modules/releasenotes/steps/1/src
update_SDK
http://10.100.98.90/tests/1017/modules/releasenotes/steps/1/src
update_HA+GEO
http://10.100.98.90/tests/1022/modules/releasenotes/steps/1/src

### CODE REVIEW

**Comment by sysrich** [comment]
I wouldn't merge these tests, as I think they allow incorrect behaviour

eg. http://10.100.98.90/tests/1019/modules/releasenotes/steps/2
Isn't this bug? you shouldn't have two sets of release notes when you're only installing one product..

http://10.100.98.90/tests/1018/modules/releasenotes/steps/5
Another bug? where is the SDK release notes - the whole point of this test is making sure that customers can get the release notes of the products they're installing

**Comment by dzedro** [comment]
fixed, but we should wait for https://bugzilla.suse.com/show_bug.cgi?id=935599

GNOME:
http://10.100.98.90/tests/1029/modules/releasenotes/steps/1/src
SDK+HA:
http://10.100.98.90/tests/1030/modules/releasenotes/steps/1/src
SDK+HA+GEO
http://10.100.98.90/tests/1031/modules/releasenotes/steps/1/src
update_SDK
http://10.100.98.90/tests/1033/modules/releasenotes/steps/1/src
update_HA+GEO
http://10.100.98.90/tests/1032/modules/releasenotes/steps/1/src

**Comment by sysrich** [comment]
If you set the test flags so the test is not important, then we wouldn't need to wait..I dont think release notes errors are worthy of always being fatal :)

**Comment by dzedro** [comment]
like this ? :) http://dzedro.qa.suse.cz/tests/1056
no more fatal flag
thx @aaannz for help

**Comment by sysrich** [comment]
Exactly :) wonderful ;)

File: tests/installation/releasenotes.pm [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+sub run(){
+    my $self=shift;
+
+    assert_and_click 'release-notes-button';    # open release notes window
+    assert_and_click 'release-notes-tab-sle';   # click on first SLES tab
+    if (get_var("ADDONS")) {
+        foreach $a (split(/,/, get_var('ADDONS'))) {
+            send_key 'alt-s';   # jump to first tab
+            $self->key_round("release-notes-tab-$a", 'right');
```
Comment (lnussel) [inline tests/installation/releasenotes.pm]: meanwhile we have send_key_until_needlematch()

**Pull request merged**